sounds/views.py

```diff
@@ -125,10 +127,18 @@ def remixed(request):
 
 
 def random(request):
-    sound_id = Sound.objects.random()
-    if sound_id is None:
+    sounds = get_random_sounds_from_solr(1)
+    sound_obj = None
+    if sounds:
+        try:
+            sound_obj = Sound.objects.get(id=sounds[0]['id'])
+        except ObjectDoesNotExist:
+            pass
+    if sound_obj is None:
+        sound_id = Sound.objects.random()
```

Comment:
This method has been updated to return an object, not an id: https://github.com/MTG/freesound/blob/b66b51d7c5294eca737fcc44f3c55aad1e786c33/sounds/models.py#L103

It'd be nice to have a test for this view which we know would catch this change.